docs/examples/basic_meter/basic_metrics.py

```diff
@@ -90,6 +95,14 @@ def usage(argv):
 staging_labels = {"environment": "staging"}
 testing_labels = {"environment": "testing"}
 
+# Views are used to define an aggregation type to use for a specific metric
+counter_view = View(requests_counter, CounterAggregator)
```

Comment:
Can you share an example of registering multiple of the same counter and aggregation, but with different labels.?

Reply:
Right now view configuration (key dropping, etc) is not implemented yet so specifying different labels for a View would not affect what is aggregated.

Reply:
Added examples.